src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/Memcached/MemcachedHelpers.cs

```diff
@@ -27,7 +26,7 @@ public static ConnectionInfo GetConnectionInfo(string key, object target, IAgent
         {
             if (_hasGetServerFailed)
             {
-                return new ConnectionInfo(DatastoreVendor.Memcached.ToKnownName(), null, -1, null);
+                return new ConnectionInfo(null, -1, null);
```

Comment:
Do we need to undo these changes? These new changes are different than what's on the `main` branch.

Reply:
No, this is related to a fix I made in the POC branch but isn't in `main` yet. The DB name has been moved somewhere more appropriate.

Reply:
The POC branch for DynamoDB removed the datastore vendor name from the `ConnectionInfo` class.  The new Memcached instrumentation was created and added to main after the DynamoDB POC branch was created.  This change is making the Memcached code align with the new way of doing things on this feature branch.